microsoft-edge/devtools-guide-chromium/sample-code/sample-code.md

```diff
@@ -90,13 +144,87 @@ This is the source code repo and its directory which stores the files for this d
+
+To download a single directory of the Demos repo:
+
+1. Go to [https://download-directory.github.io/](https://download-directory.github.io/) and then paste the URL [https://github.com/MicrosoftEdge/Demos/tree/main/demo-to-do](https://github.com/MicrosoftEdge/Demos/tree/main/demo-to-do).  
+
+   The `.zip` file is placed in your download directory.  Unzip these webpage source files into a suitable location.
+
+
+#### Clone the Demos repo
+
+Cloning a repo enables updating your local copy when the repo is updated.  The GitHub UI and various tools support cloning.  We'll show cloning by using Visual Studio Code, but you can use many other coding tools, such as GitHub Desktop, Visual Studio, or git bash shell.
+
+To clone the repo:
+
+1. In a new window or tab, go to the [MicrosoftEdge / Demos](https://github.com/MicrosoftEdge/Demos#readme) repo.
+
+1. If the green **Code** button isn't shown, click **Demos** in the path **Microsoft Edge** / **Demos** in the upper left to go to the main page of the repo.
+
+1. Click the **Code** dropdown button, and then click the **Copy** button next to the URL **https://github.com/MicrosoftEdge/Demos.git**.  You can then paste the URL into git bash or a Visual Studio Code dialog, for example.
+
+   Or, click the **Code** dropdown button, and then click **Open with Visual Studio** if it appears.  A list of **Handler Selector** items is offered, one per Visual Studio instance installed.  This option appears only if you're logged in.
+
+   ![Cloning the MicrosoftEdge/Demos repo](./sample-code-images/clone-repo.png)
+
+1. In Visual Studio Code, in the Activity Bar, click the **Source Control** (![Source Control icon](./sample-code-images/source-control-icon.png)) button, and then click the **Clone Repository** button.
+
+1. In the **Provide repository URL** text box, paste the copied URL: **https://github.com/MicrosoftEdge/Demos.git** and then press `Enter`.  A folder selection dialog opens.
+
+   ![The Clone Repository button in Visual Studio Code](./sample-code-images/clone-repository-button.png)
+
+1. Navigate to your desired path, such as `C:\Users\username\Documents\GitHub` or `Users/username/GitHub`, and then click the **Select Repository Location** button.
+
+1. The message **Cloning git repository** appears, then you're prompted to open the cloned repository.  Click the **Open** button:
+
+   ![Prompted to open the cloned repo](./sample-code-images/prompt-open-cloned-repo.png)
+
+1. If prompted **Do you trust...**, click the **Yes** button.  Or, click the **No** button and continue with most parts of this walkthrough.
+
+   The **Explorer** tree lists many demos, including **demo-to-do**.
+
+1. In Visual Studio Code, select **File** > **Close Folder**.
+
+1. In the Activity Bar, click the **Microsoft Edge Tools** (![Microsoft Edge Tools icon](./sample-code-images/microsoft-edge-tools-icon.png)) button.  The **Microsoft Edge Tools** pane opens.
```

Comment:
I may have missed it, but I don't think we're telling people to install the Microsoft Edge Tools extension before. So they won't have this icon in the Activity bar.

Also, this section is about cloning the repo, not using the Edge tools in VS Code. So I don't think this part is necessary.

Reply:
reworked and expanded to give more guidance on what you can to with the local Demos repo to open and edit files.  Body doesn't mention the extension now.  See also links, instead.